Remove s2i (#1344)
3scale/APIcast: doc/build.md

## Components

APIcast is using [docker](https://www.docker.com/) to build the APIcast runtime image.

> You'll need to have docker installed and available on your system.

Reviewer: ```suggestion
You must have docker installed and available on your system.
```

The development image used is based on [UBI](https://developers.redhat.com/products/rhel/ubi).
It builds on heavy OpenShift base images.

> In the future, we would like to leverage multistage dockerfiles and use a very minimal runtime image.

Reviewer: ```suggestion
In future releases, we will leverage multistage docker files and use a minimal runtime image.
```

## Build process

**runtime image**

> The runtime image build is defined in the `Makefile`. The `make runtime-image` is meant for the upstream release build.

Reviewer: ```suggestion
The runtime image build is defined in the `Makefile`. The `make runtime-image` is for the upstream release build.
```
